plaso/storage/interface.py

```diff
@@ -22,7 +22,7 @@ class BaseStore(object):
   Attributes:
     format_version (int): storage format version.
     serialization_format (str): serialization format.
-    storage_type (str): storage type.
+    storage_typg (str): storage type.
```

Comment:
I wonder what sequence of vi shortcuts would make that typo =)

Reply:
likely more a factor of me randomly hitting the keyboard at some strange time of day, changed